### DISCUSSION TITLE
Generic ERP Connector only work with Postgres SQL?

### DISCUSSION THREAD

**Discussion by Drayang** [frePPLe/frepple#479] [Q&A] [answered]
Since Frepple is using Postgres SQL database, any the ERP system i want to integrate with is using mariadb SQL, may i ask whether the connector can work if both frepple and erp system are using different database? Or the connector is specifically design to a ERPsystem that is also employed Postgres SQL database? Thank you!

**Comment by jdetaeye** [comment, marked as answer]
Yes, it's possible to connect to mariadb.   The generic erp connector can connect to any data source & technology that is accessible with python.

As you mention, the frepple database itself is always on postgresql.
